packages/fast-color-explorer/README.md

```diff
@@ -0,0 +1,16 @@
+# Introduction 
+Application for UI testing and development.
+
+# Getting Started
```

Comment:
if you use a single "#" these will all show as the equivalent of h1, so this, and the ones below, should be ## or greater.